src/test/java/yevhenii/lostfilmdemo/LostfilmDemoApplicationTests.java

```diff
@@ -1,13 +1,30 @@
 package yevhenii.lostfilmdemo;
 
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import yevhenii.lostfilmdemo.repository.TVSeriesRepository;
+import yevhenii.lostfilmdemo.services.FeedService;
+import yevhenii.lostfilmdemo.services.TVSeriesService;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 @SpringBootTest
 class LostfilmDemoApplicationTests {
 
+    @Autowired
+    FeedService feedService;
+    @Autowired
+    TVSeriesService service;
+    @Autowired
+    TVSeriesRepository repository;
+
     @Test
     void contextLoads() {
+
+        assertThat(service).isNotNull();
```

Comment:
what this test is doing? you will never get null here, in case if you have some issue with those beans context will fail to start and test will fail anyway